components/brave_wallet_ui/common/hardware/ledgerjs/ledger-messages.ts

```diff
@@ -4,18 +4,30 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { loadTimeData } from '../../../../common/loadTimeData'
+import type {
+  SolLedgerFrameCommand,
+  SolLedgerFrameResponse
+} from './sol-ledger-messages'
+import type {
+  EthLedgerFrameCommand,
+  EthLedgerFrameResponse
+} from './eth-ledger-messages'
 
 export const LEDGER_BRIDGE_URL = loadTimeData.getString('braveWalletLedgerBridgeUrl').slice(0, -1) // Strip off trailing '/' in URL
 
```

Comment:
I just realised it's not part of this PR 😄 Still, would be good to add a check.

Reply:
I'd assume not because we normally prefix constants with k in the codebase, so it's probably coming as an input from somewhere.

Reply:
It's hardcoded here https://github.com/brave/brave-core/blob/4b221b3083f4b13ee75918d555bcb3b91fb4cbf2/components/constants/webui_url_constants.cc#L41

I'm not sure why exactly, but I'm pretty sure the feature doesn't work at all if the trailing slash is omitted there, otherwise I would have opted to omit it when hardcoding.  So we should be able to be confident that it will always be there.

Ideally I would not use `slice` and instead do something like this 
```
new URL(loadTimeData.getString('braveWalletLedgerBridgeUrl')).origin
```
however that makes the tests fail (even if you specifically define `braveWalletLedgerBridgeUrl`, in the test data https://github.com/brave/brave-core/blob/4b221b3083f4b13ee75918d555bcb3b91fb4cbf2/components/test/testPolyfills.ts#L24-L26).  

Eventually I gave up and did this.

Reply:
@stoletheminerals updated here https://github.com/brave/brave-core/pull/14373/commits/38b9e55814f7952fd8ec987c42932b01e99e121a

Reply:
Seems like a good defensive fix for this